command/agent.go

```diff
@@ -259,6 +260,18 @@ Telemetry Options:
   -telemetry-circonus-broker-select-tag
     A tag which is used to select a broker ID when an explicit broker ID is not
     provided.
+  
+High Availability Options:
```

Comment:
Do we want to support the other config parameters on the CLI also?

Reply:
They are defined, I forgot to update the docs, let me do it